servicelayer/worker.py

```diff
@@ -22,26 +29,29 @@
 INTERVAL = 2
 TASK_FETCH_RETRY = 60 / INTERVAL
 
-TASK_STARTED = Counter(
-    "task_started_total",
+REGISTRY.unregister(GC_COLLECTOR)
+REGISTRY.unregister(PROCESS_COLLECTOR)
```

Comment:
Are these not useful to us?

Reply:
I’ve removed the process collector as the metrics it provides are probably a bit redundant with regards to the metrics you usually already collect in a containerized environment (CPU time, memory usage, …)

With regards to garbage collection, I’m honestly not sure -- this is usually something I don't mess with in Python.

Both collectors don’t work well in multi-process mode (as they are per-process metrics), so they are not available for the API service, but I’m happy to add them back for the Aleph and ingest-file workers.

Reply:
Thanks for explaining this!